src/test-e2e/po/stepper.po.ts

```diff
@@ -28,7 +28,11 @@ export class StepperComponent extends Component {
   }
 
   next() {
-    return this.nextButton().click();
+    const nextButton = this.nextButton();
+    return browser.controlFlow()
```

Comment:
Do you need to do browser.controlFlow.execute - surely browser.executeScript runs in the control flow?